automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/FireCloudView.scala

```diff
@@ -1,6 +1,6 @@
 package org.broadinstitute.dsde.firecloud.page
 
-import org.broadinstitute.dsde.workbench.util.WebBrowserUtil
```

Comment:
TODO non-blocker: Remove most of the `page` classes from Leonardo

Reply:
can we github ticket these todos?

Reply:
yep I'll write something up